src/main/java/dk/kb/storage/storage/DsStorage.java

```diff
@@ -787,7 +788,9 @@ public int updateMTimeForRecord(String recordId) throws Exception {
             stmt.setLong(1, nowStamp);      
             stmt.setString(2, recordId);
            int numberUpdated =  stmt.executeUpdate();           
-           return numberUpdated;
+           RecordsCountDto modifiedDto= new RecordsCountDto();
+           modifiedDto.setCount(numberUpdated);
+           return modifiedDto;
```

Comment:
You could just call this countDto as you do other places

Reply:
fixed.